engines/bladerunner/script/scene/ug15.cpp

```diff
@@ -40,7 +41,8 @@ void SceneScriptUG15::InitializeScene() {
 	} else {
 		Setup_Scene_Information(-238.0f, 48.07f,  222.0f, 180);
 		if (Game_Flag_Query(kFlagUG15RatShot)
-		 && Random_Query(1, 10) == 10
+		 && (Random_Query(1, 10) == 10)
+		 && !(_vm->_cutContent && _vm->_settings->getDifficulty() == kGameDifficultyEasy)
```

Comment:
Actually, could you replace your _vm->_settings->getDifficulty() with Query_Difficulty_Level() in the file?
And then you can remove the "#include "bladerunner/settings.h"